serialization/typescript/json/tsconfig.cjs.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "./tsconfig.base.json",
```

Comment:
node12, which gets [out of support in April](https://nodejs.org/en/about/releases/) already supports 99% of the es6 spec, including es6 modules.
about [95% of the browsers out there support](https://caniuse.com/es6) es6 at the time of writing.
we should be able to ship es6 code with es6 modules for everyone, and tell people complaining about compatibility to patch things up with transpilation + polyfills on their end.
I strongly suggest we DO NOT maintain an es5 version, it adds a lot of complexity for very few use cases.